sycl/source/detail/windows_pi.cpp

```diff
@@ -0,0 +1,11 @@
+#include <windows.h>
+#include <winreg.h>
+#include <string>
+
+void *loadOsLibrary(const std::string &PluginPath) {
+  return (void *)LoadLibraryA(PluginPath.c_str());
```

Comment:
Would it be useful to prevent Windows error dialogue boxes by using [`SetErrorMode`](https://docs.microsoft.com/en-gb/windows/win32/api/errhandlingapi/nf-errhandlingapi-seterrormode), e.g., to be able to run on headless nodes?

Reply:
The latest [PI doc PR]() writes that the `LIB` environment variable is used to find dynamic libraries. Though looking into the description of [LoadLibraryA](https://docs.microsoft.com/en-us/windows/win32/api/libloaderapi/nf-libloaderapi-loadlibrarya) and [Dynamci-Link Library Search Order](https://docs.microsoft.com/en-gb/windows/win32/dlls/dynamic-link-library-search-order#standard-search-order-for-desktop-applications) I do not see this env var mentioned. Is the plan to implement use of the `LIB` env var here to not go through the `PATH`?